data_dir/c-s_lwt_data_not_for_update.yaml

```diff
@@ -0,0 +1,63 @@
+# Based on https://gist.github.com/tjake/8995058fed11d9921e31
```

Comment:
Please change all comments in this file, explain in the header what is the purpose of this file and how it's different from the other file.

Reply:
remove this file